src/routes/governance/hooks/use-vote-information.ts

```diff
@@ -168,8 +168,13 @@ export const useVoteInformation = ({
     yesPercentage,
     noTokens,
     yesTokens,
+    yesVotes: new BigNumber(proposal.votes.yes.totalNumber),
+    noVotes: new BigNumber(proposal.votes.no.totalNumber),
+    totalVotes: new BigNumber(proposal.votes.yes.totalNumber).plus(
+      proposal.votes.no.totalNumber
```

Comment:
You've wrapped `proposal.votes.yes.totalNumber` with BigNumber and called `plus` but not wrapped `proposal.votes.not.totalNumber` with BigNumber.

Does `bignumber.js` automatically construct a BigNumber object with the argument passed to `plus`? Or do we need to do it explicitly?

Reply:
It auto-casts 👍